dashboard/lib/user_multi_auth_helper.rb

```diff
@@ -88,6 +88,7 @@ def migrate_to_multi_auth
         end
     end
     self.provider = 'migrated'
+    clear_single_auth_fields
     save
```

Comment:
There's an implied save in `clear_single_auth_fields` - worth removing the redundancy, or no big deal because ActiveRecord probably does the right thing when there's nothing to save?

Reply:
actually, looking at this closer I think it's worth it to remove this method entirely